backend/ops_api/ops/schemas/change_requests.py

```diff
@@ -0,0 +1,37 @@
+from marshmallow import EXCLUDE, Schema, fields
+
+from ops_api.ops.schemas.users import SafeUserSchema
+
+
+class ChangeRequestSchema(Schema):
+    id = fields.Int(required=True)
+    type = fields.Str(required=True)
+    display_name = fields.String(required=True)
+    status = fields.String(required=True)
+    requested_change_data = fields.Dict(required=True)
+    requested_change_diff = fields.Dict(required=True)
+    managing_division_id = fields.Int(required=True, default=None, allow_none=True)
+    created_by = fields.Int(required=True)
+    created_by_user = fields.Nested(SafeUserSchema(), default=None, allow_none=True)
+    created_on = fields.DateTime(required=True)
```

Comment:
Please make sure that the client HTTP request cannot set the created_by, created_on, updated_by, updated_on - these should only be set in the API service/DB

Reply:
These are only for responses, there's no way to POST/PATCH change requests themselves.  Maybe I'll just tweak the naming to make that more obvious.